src/protocols/ethereum/libs/EtherscanService.ts

```diff
@@ -94,12 +95,16 @@ export class EtherscanService {
       offset?: string;
       /** Decides if returned transaction list will contain all ERC-20 tokens or one specified */
       assetContractId?: AssetContractId;
+      startblock?: string;
+      endblock?: string;
     } = {},
   ): Promise<ITransaction[]> {
     const {
       assetContractId,
       page = '1',
       offset = TXS_PER_PAGE.toString(),
+      startblock = '0',
```

Comment:
No need to add `startblock` and `endblock` here.

Reply:
The comment is about list of all transactions. No need to specify `startblock` and `enblock` there.